packages/manager/modules/exchange/src/account/mfa/account-mfa.html

```diff
@@ -0,0 +1,32 @@
+<div data-ng-controller="ExchangeAccountMfaCtrl as $ctrl">
```

Comment:
@cbourgois No component possible here ?

Reply:
*echo*

Reply:
We could create a component for that but it will not be a _real_ component.

This component will not take parameters (_bindings_), because it needs to use _statefull_ Exchange services to retrieve the current account and the current exchange service.
IMO, it would be necessary to modify the Exchange module in depth for this to bring more sense.